### PR TITLE
feat: Create bot flow

### LINKED ISSUE #1728: UX | Create-new wizard extension and enhancement
Pick up where we left off with the Create New bot/dialog wizard.

Business justification:
Currently, Composer skips the step if Design/Plan and jumps right into building a dialog, assuming the user knows what they are creating and why, and that they are fluent with Composer and the SDK. The 'Created New' wizard attached below helps people getting started by asking them to input some information in a 'human' /conversational manner, and uses this to bootstrap the dialog. I believe this will lower the threshold for users to get started with Composer and hit the ground running and making it easier to understand concepts such as LG, LU, Events, Triggers etc. as they are already repopulated with the content provided in the Wizards.

Also 
- We are approaching the stage where we're allowing people to create multiple types of dialogs, so our Create New flow needs to scale to support this and make this easily discoverable:
- adaptive dialogs & sub-dialogs
- generated dialogs
- QnA dialogs
- Samples

https://www.figma.com/file/NqlONHOHjE0amtoHlLlopg9p/Composer-XY-07.16?node-id=1632%3A25465

Enable users to 'seed' their dialog with 
- Take full advantage of Managed LU

### PR DESCRIPTION
**Pull request by liweitian**
## Description
Applied new design for create bot option 3.

## Task Item
Closes #1728 
## Screenshots
aaaaa

### CODE REVIEW

**Comment by cwhitten** [comment]
@liweitian can you please add some tests and also address the currently failing Unit and E2E tests? Thank you.

File: Composer/packages/client/src/CreationFlow/CreateOptions/index.js [hunk 1]
```diff
@@ -27,83 +47,120 @@ export function CreateOptions(props) {
...
+      minWidth: 200,
+      maxWidth: 450,
+      isResizable: !disabled,
+      data: 'string',
+      styles: rowTitle(disabled),
+      onRender: item => item.description,
+    },
+    {
+      key: 'column3',
+      name: formatMessage('Created'),
```
Comment (cwhitten) [inline Composer/packages/client/src/CreationFlow/CreateOptions/index.js:92]: This column isn't necessary. Please remove

File: Composer/packages/client/src/CreationFlow/CreateOptions/index.js [hunk 2]
```diff
@@ -27,83 +47,120 @@ export function CreateOptions(props) {
...
+      fieldName: 'name',
+      minWidth: 150,
+      maxWidth: 200,
+      isResizable: !disabled,
+      data: 'string',
+      styles: rowTitle(disabled),
+      onRender: item => item.name,
+    },
+    {
+      key: 'column2',
```
Comment (cwhitten) [inline Composer/packages/client/src/CreationFlow/CreateOptions/index.js:80]: can you give this field a key of "description"

File: Composer/packages/client/src/CreationFlow/CreateOptions/index.js [hunk 3]
```diff
@@ -27,83 +47,120 @@ export function CreateOptions(props) {
...
+    if (option === 'CreateFromTemplate') {
       onNext(template);
     } else {
       onNext(emptyBotKey);
     }
   };
 
+  const tableColums = [
+    {
+      key: 'column1',
```
Comment (cwhitten) [inline Composer/packages/client/src/CreationFlow/CreateOptions/index.js:69]: can you give this field a key of "name"

File: Composer/packages/client/src/CreationFlow/CreateOptions/index.js (hunk 1 above)
Comment (liweitian) [reply Composer/packages/client/src/CreationFlow/CreateOptions/index.js]: Removed. But the design has this column.

File: Composer/packages/client/src/CreationFlow/CreateOptions/index.js [hunk 4]
```diff
@@ -27,83 +47,109 @@ export function CreateOptions(props) {
...
+      );
+    }
+    return null;
+  };
   return (
     <Fragment>
       <ChoiceGroup
         label={formatMessage('Choose how to create your bot')}
-        defaultSelectedKey="Create from scratch"
+        defaultSelectedKey="CreateFromScratch"
```
Comment (a-b-r-o-w-n) [inline Composer/packages/client/src/CreationFlow/CreateOptions/index.js:113]: Its probably a good idea to make this and `CreateFromTemplate` a const value.

File: Composer/cypress/integration/CreateNewBot.spec.ts [hunk 5]
```diff
@@ -21,7 +21,7 @@ context('Creating a new bot', () => {
 
   it('can create a bot from the ToDo template', () => {
     cy.findByTestId('Create from template').click();
-    cy.findByTestId('TodoSample').click();
+    cy.findByTestId('TodoSample').click({ force: true });
```
[the remark this answers is not in the document]
Comment (liweitian) [reply Composer/cypress/integration/CreateNewBot.spec.ts:24]: https://github.com/cypress-io/cypress/issues/3450
https://github.com/cypress-io/cypress/issues/871

**Pull request merged**